### PR TITLE
Payment status page and issue fixes

### PR DESCRIPTION
**Pull request by alex-dhanasekar**
Payment status page and Hotdeals section title font size became too small issue fix

--Updated screenshots

image
image
image

### CODE REVIEW

File: config/HomeConfig.js [hunk 1]
```diff
@@ -115,10 +116,10 @@ const HotDealsData = {
   title: "HOT DEALS",
   linkText: "VIEW ALL",
   OfferText: (
-    <span>
+    <React.Fragment>
```
Comment (Sumit-Kap) [inline config/HomeConfig.js:119]: FYI you can also use <></> another alternative of react.fragment.

File: pages/PaymentStatus/index.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import Header from "../../components/Header";
+import Footer from "../../components/Footer";
+import UIBUtton from "../../styles/UIKit/Button";
+import PaymentCompleteConfig from "../../config/PaymentCompleteConfig";
+
+export default class PaymentStatus extends Component {
+  static async getInitialProps({ query }) {
```
Comment (Sumit-Kap) [inline pages/PaymentStatus/index.js:11]: is this server side rendered?

**Review by Sumit-Kap — APPROVED** [review]

File: pages/PaymentStatus/index.js (hunk 2 above)
Comment (alex-dhanasekar) [reply pages/PaymentStatus/index.js:11]: Yes Sumit, Since all pages are rendered using nextjs only.

File: config/HomeConfig.js (hunk 1 above)
Comment (alex-dhanasekar) [reply config/HomeConfig.js:119]: I tried doing that Sumit but sometimes the js syntax highlighting doesn’t work properly in my vs code when i used <></>

**Pull request merged**